Header header
Servo   pan
Servo   tilt